tests/test_connector_facebook.py

```diff
@@ -7,8 +7,7 @@
 from opsdroid.__main__ import configure_lang
 from opsdroid.core import OpsDroid
 from opsdroid.connector.facebook import ConnectorFacebook
-from opsdroid.message import Message
-from opsdroid.__main__ import configure_lang
+from opsdroid.events import Message
```

Comment:
Are the changes in this file a rebase fail?

Reply:
Very likely. I've actually also realised that it may be necessary to change the ordering of arguments in the `Event` subclasses' call signatures, so everything else should carry on using `message.Message` for now anyway because otherwise stuff will break.